README.md

```diff
@@ -46,8 +62,140 @@ The main bot will handle all inputs and ouputs from the user.
 
 See in-depth documentation [here](app/greetings/README.md).
 
-## Product & Store Mini-bot
+## Product & Store Mini-bots
 
 This mini-bot will handle any question related to product price, stock, and store information. The bot first determines what topic is of interest (product or store information) and breaks the user's message down into keywords. The bot then compares these words and checks them in order to create the most appropriate response, and returns the response to the main bot.
 
-See in-depth documentation [here](app/products/README.md).
\ No newline at end of file
+See in-depth documentation [here](app/products/README.md).
+
+## Tests
+We are using pytest for all of our tests. Our test cases can be found in the `test` folder.
+
+### Test all cases:
+```console
+python -m pytest
+```
+
+### Test selected cases:
+
+- database
+- store_info
+- prod_info
+- intentDetectionTest
+
+```console
+python -m pytest -v -m <selected case>
+```
+
+## API
+You can call the API from any other Python script to check store's information or products information. This does not require the bot to be running or a diagflow key.
+
+**Make sure to import the correctly, as the API is not imported by default, and is dependent on your project's file hierarchy.**
+
+### Store Info Bot API
```

Comment:
Since the API is separated from the bot, removing the word `Bot` would be clearer that user is not using the bot.